references/backup.md

```diff
@@ -0,0 +1,58 @@
+# Backup
+
+The `backup` route allow user to trigger a backup creation and download created backup.
+
+## Trigger backup
+
+<RouteHighlighter method="POST" route="/backup"/>
+
+Trigger a backup creation process
```

Comment:
```suggestion
Trigger the backup creation process.
```
When using `a` you imply that there are more than one type of `backup creation processes`. Which I don't think is the case. Correct my if i'm wrong.